delta/plugins/composite-views/src/test/resources/routes/responses/view.json

```diff
@@ -0,0 +1,102 @@
+{
+  "@context": [
+    "https://bluebrain.github.io/nexus/contexts/composite-views.json",
+    "https://bluebrain.github.io/nexus/contexts/metadata.json"
+  ],
+  "@id": "https://bluebrain.github.io/nexus/vocabulary/{{uuid}}",
+  "@type": [
+    "View",
+    "CompositeView"
+  ],
+  "sources": [
+    {
+      "@id": "http://example.com/cross-project-source",
+      "project": "org/otherproject",
+      "identities": [
+        {
+          "@id": "http://localhost/v1/anonymous",
+          "@type": "Anonymous"
+        }
+      ],
+      "resourceSchemas": [
+      ],
+      "resourceTypes": [
+      ],
+      "includeDeprecated": false,
+      "@type": "CrossProjectSource",
+      "_uuid": "{{uuid}}"
+    },
+    {
+      "@id": "http://example.com/remote-project-source",
+      "project": "org/remoteproject",
+      "resourceSchemas": [
+      ],
+      "resourceTypes": [
+      ],
+      "includeDeprecated": false,
+      "@type": "RemoteProjectSource",
+      "endpoint": "http://example.com/remote-endpoint",
+      "_uuid": "{{uuid}}"
+    },
+    {
+      "@id": "http://example.com/project-source",
+      "resourceSchemas": [
+      ],
+      "resourceTypes": [
+      ],
+      "includeDeprecated": false,
+      "@type": "ProjectSource",
+      "_uuid": "{{uuid}}"
+    }
+  ],
+  "projections": [
+    {
+      "@id": "http://example.com/blazegraph-projection",
+      "query": "SELECT * WHERE {?s ?p ?p}",
```

Comment:
select queries should fail, shouldn't they?

Reply:
yes, this query is not correct, I just wanted to have something short to keep the example responses/requests short. I can change it for another query.

Reply:
I've created an issue for the query bit: https://github.com/BlueBrain/nexus/issues/2220